### PR TITLE
Simpler tarball preparation

### PR DESCRIPTION
**Pull request by mvidner**
Depends on https://github.com/libyui/libyui/pull/92

### CODE REVIEW

File: package/libyui-qt.spec [hunk 1]
```diff
@@ -34,50 +34,56 @@ BuildRequires:  pkgconfig(Qt5Gui)
...
 %description
-@DESCRIPTION@
+This package contains the Qt user interface
+component for libYUI.
 
-%package -n @PROJECTNAME@@SONAME_MAJOR@
 
-Requires:       lib@BASELIB@@SONAME_MAJOR@
-Provides:       lib@BASELIB@-qt = %{version}
+%package -n libyui-qt7
```
Comment (jreidinger) [inline package/libyui-qt.spec]: spreading 7 over file is not nice, I think some macro that define number will be better

**Comment by nilxam** [comment]
any update here?

**Comment by jreidinger** [comment]
@nilxam martin is currently sick. I hope he will work on it after return to work.

**Comment by nilxam** [comment]
oh, I see. no worries. Get well soon Martin.

**Pull request merged**